### DISCUSSION TITLE
Scripting Error when opening Blocks.Discrete

### DISCUSSION THREAD

**Discussion by tburch-LSS** [OpenModelica/OpenModelica#13220] [Q&A] [answered]
i noticed a scripting error whenever i open Modelica.Blocks.Discrete in OMEdit (v1.24.1) on Ubuntu 22.04:

[3] 17:38:26 Scripting Error
[/home/rroy/.openmodelica/libraries/Modelica 4.0.0+maint.om/Blocks/Discrete.mo:396:11-415:48:writable] Error: Function Text has no input parameter named color.

**Comment by perost** [comment, marked as answer]
The error is correct, it should be `textColor` and not `color`. This is fixed in the upcoming MSL 4.1.0.

**Reply by tburch-LSS** [reply]
when it comes to errors, "correct" ones are the best kind... ;-)
i suspected this had to be a modelica problem. thanks!
